transactions.go

```diff
@@ -362,6 +397,48 @@ func GetWalletNonce(provider *ethrpc.Provider, walletConfig WalletConfig, wallet
 	return nonceResult, nil
 }
 
+func IsNonceChangedEvent(logs []*types.Log) bool {
+	if len(logs[0].Topics) != 1 {
+		return false
+	}
+	if logs[0].Topics[0] != NonceChangeEventSig {
+		return false
+	}
+	return true
+}
+
+// prepareTransactionsForEncoding checks the transactions data structure with basic
+// integrity checks
+func prepareTransactionsForEncoding(txns Transactions) (Transactions, error) {
```

Comment:
Can you remove this and use `Transactions.EncodedTransactions()` instead? It was renamed on master.